### DISCUSSION TITLE
Push to other private repository in codespace

### DISCUSSION THREAD

**Discussion by MFAshby** [community/community#5254] [Codespaces] [answered]
Hi, 

I have a codespace set up with multiple git repositories. I have started with one repository, and then cloned a second using the git:clone action and selecting the repository from github 

![image](https://user-images.githubusercontent.com/1760515/130517336-4cc36f67-b887-47b1-b7ad-a4c12384bb03.png)

I have then added the folder to my workspace when prompted, and I'm able to browse and edit files and make commits locally. Great!

![image](https://user-images.githubusercontent.com/1760515/130517458-bcadd1bb-a698-499f-80ce-da9d7cc5b742.png)

However, I then try to push my changes to this second repository:

![image](https://user-images.githubusercontent.com/1760515/130517606-9ec3e85a-b5c9-46b3-bfea-981cdb423bf3.png)

And I get the following error:

![image](https://user-images.githubusercontent.com/1760515/130517729-79849628-f817-4d62-854e-c49f4baa8e6b.png)

And the command output is: 
```
> git push origin phr-8766:phr-8766
remote: Repository not found.
fatal: repository 'https://github.com/patientsknowbest/docker.git/' not found
```

Please help! I have a multi-repo project, and this is making codespaces unusable for tasks which need changes in multiple repos.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @MFAshby, thanks for reaching out. 

Here's an example on how to use Codespaces in a multi-repo scenario: https://github.com/Chuxel/codespaces-multi-repo

We have an item on our roadmap to make this easier, hope this workaround helps until then :)

**Reply by MFAshby** [reply]
that works, thanks very much. Looking forward to this being a bit more convenient in future though.
